### PR TITLE
Include file for ASP.NET Core 2.1

### LINKED ISSUE #5703: Create 2.1 pre-release include
per `@danroth27`

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #5703 
Once we agree on the verbage, I'll ask  `@danroth27` to review.

### CODE REVIEW

**Comment by rachelappel** [comment]
Isn't there already a message about this at the top of the docs? Is this to replace that?

**Comment by Rick-Anderson** [comment]
@danroth27  please 30 second review of 2.1 preview include file

@rachelappel 

>Isn't there already a message about this at the top of the docs? Is this to replace that?

Where?

**Comment by Rick-Anderson** [comment]
@danroth27  30 second review of 2.1 preview "warning header"

File: aspnetcore/includes/2.1.md [hunk 1]
```diff
@@ -0,0 +1 @@
+ASP.NET Core 2.1.0-preview1 is in preview and not recommended for production use.
```
Comment (danroth27) [inline aspnetcore/includes/2.1.md]: How about: ASP.NET Core 2.1 is in preview and not recommended for production use at this time.

**Pull request merged**